python/ql/src/semmle/python/objects/Descriptors.qll

```diff
@@ -20,13 +20,39 @@ class PropertyInternal extends ObjectInternal, TProperty {
         this = TProperty(_, _, result)
     }
 
+    private CallNode getCall() { this = TProperty(result, _, _) }
```

Comment:
I guess there's no particular reason to make this `private`. Also, perhaps `getCallNode` is a better name for this, since `Call` already has a meaning in our libraries.

Reply:
I recall not being overly enthusiastic about our support for `property`, so I made my best effort to make it easy to rewrite it completely later on :smile:

I changed the name though :+1: if you feel strongly against `private` I can change it as well

Reply:
Okay, keep it `private` for now.